tools/gmhazard_scripts/db_creation/empirical_db/new_calc_emp_ds.py

```diff
@@ -0,0 +1,253 @@
+import argparse
+import time
+import math
+
+import pandas as pd
+import numpy as np
+
+import common
+import gmhazard_calc as gc
+from empirical.util import empirical_factory, classdef, openquake_wrapper_vectorized
```

Comment:
Import the module pls

Reply:
Surprisingly they are modules haha
![image](https://user-images.githubusercontent.com/7849113/160963939-21eb9e37-f399-441e-8765-6a333f295637.png)